### PR TITLE
Added documentation for large data sets

### PR DESCRIPTION
**Pull request by levimcg**
This PR add new examples to the pagination component including how to handle a large amount of pages.

### CODE REVIEW

File: content/components/navigation/pagination.md [hunk 1]
```diff
@@ -5,25 +5,125 @@ description: "The pagination component is used to break up large sets of data ac
...
+      <a href="#" aria-label="Page 5">5</a>
+    </li>
+  </ul>
+</nav>
+{{< /example >}}
+
+## Large data sets
+When you need to paginate large sets of data over many pages, use a modified pagination component shown in the following example.
+
+- Use the ellipsis icons to help visually indicate that there are more sets of pages before and/or after the currently displayed set. Note in this example we use `tabindex` `aria-hidden` attributes to remove the ellipsis from the tab order and hide the content from screen readers
```
Comment (illusivesunrae) [inline content/components/navigation/pagination.md]: > Note in the this example we use `tabindex` `aria-hidden` attributes...

> ...we use the `tabindex` and `aria-hidden` attributes...

File: content/components/navigation/pagination.md [hunk 2]
```diff
@@ -45,50 +145,69 @@ When using the `.is-disabled` classes to create pagination links that appear to
 ### Positioning
 The pagination component is left-aligned by default, but you can easily center or right-align it using the `rvt-pagination--center` or `rvt-pagination--right` modifier classes.
```
Comment (illusivesunrae) [inline content/components/navigation/pagination.md]: I notice here that the classes are referenced without a period at the beginning, but in other places (ex. line 181) a period is still used. Not sure which way is preferred, but picking one for consistency would probably be best.

Comment (levimcg) [reply content/components/navigation/pagination.md]: Looks like Scott has already has us covered on this one. 😉 

https://github.com/indiana-university/rivet-source/wiki/Editorial-style-guide#code

I'll update that.

Comment (illusivesunrae) [reply content/components/navigation/pagination.md]: 🙌

**Pull request merged**